examples/fluids/navierstokes.c

```diff
@@ -1831,6 +1833,15 @@ int main(int argc, char **argv) {
                                restrictq, ctxSetup, ftime); CHKERRQ(ierr);
     ierr = VecNorm(Qexact, NORM_1, &norm_exact); CHKERRQ(ierr);
     ierr = VecAXPY(Q, -1.0, Qexact);  CHKERRQ(ierr);
+    // Visualize the absolute error
+    ierr = PetscSNPrintf(filepath, sizeof filepath, "%s/ns-err-%03D.vtu",
+                         user->outputfolder, user->contsteps);
+    CHKERRQ(ierr);
+    ierr = PetscViewerVTKOpen(PetscObjectComm((PetscObject)Q), filepath,
+                            FILE_MODE_WRITE, &viewer); CHKERRQ(ierr);
+    ierr = VecView(Q, viewer); CHKERRQ(ierr);
+    ierr = PetscViewerDestroy(&viewer); CHKERRQ(ierr);
+    // Compute the relative error
```

Comment:
I am not sure whether we should keep this or not. @jedbrown , let me know if you think I should drop this commit or make any changes (like visualizing the relative error instead).

Reply:
I don't think it's necessary to keep it, but if you do, please disable it by default (adding an option to activate it).

Reply:
Thanks! I dropped it.